desktop-app/app/actions/browser.js

```diff
@@ -185,6 +188,21 @@ export function toggleDeviceMuted(deviceId, isMuted) {
   };
 }
 
+export function toggleAllDevicesDesignMode(allDevicesInDesignMode) {
+  return {
+    type: TOGGLE_ALL_DEVICES_DESIGN_MODE,
+    allDevicesInDesignMode,
+  };
+}
+
+export function toggleDeviceDesignMode(deviceId, designModeOn) {
```

Comment:
Do we really need a separate dispatch function? Can we keep this and remove the dispatch?

Reply:
I just followed "the style" of the code around, but I think is ok to change it